source/license.md

```diff
@@ -1,10 +1,13 @@
 # <i class="fab fa-creative-commons"></i> License
 
-OpenFisca is free software made available under the [AGPL license](https://choosealicense.com/licenses/agpl-3.0/). This means that you are free to use, install and modify it, and that you have to contribute your changes back to the community. This is the only way a contributive digital common can be sustainable, so we are very happy that you take part in this shared effort!
+OpenFisca is a free software made available under the [AGPL license](https://choosealicense.com/licenses/agpl-3.0/). This means that you are free to use, install and modify it with the promise to contribute back your changes to the community. This is the only way a contributive digital common can be made sustainable.
+Any contributions are welcome as long as they abide by our contribution guidelines. We are happy to have you here at the OpenFisca community.
 
-## Computation results
+## OpenFisca is used as
```

Comment:
I am not sure I understand the purpose of this additional title 🙂

Reply:
The tittle was added to depict the message that " OpenFisca has many forms of usage and each case needs to be credited as follows ". I can remove it if it feels illogical

Reply:
I think unfinished sentences don't belong in titles.

Reply:
Let's remove this additional title then 🙂

Reply:
okay 👍